lib/registerPlugins.js

```diff
@@ -1,13 +1,12 @@
 'use strict';
 
-function registerPlugins(options) {
-	var plugins = options.plugins;
+var path = require('path');
 
-	if (plugins.length) {
-		plugins.forEach(function(plugin) {
-			require(plugin)(options);
-		});
-	}
+function registerPlugins(options) {
+	options.plugins = options.plugins.map(plugin => {
+		let pluginPath = path.join(process.cwd(), 'node_modules', plugin);
```

Comment:
Node wasn't looking into the application node_modules directory, but into the node_modules directory of `electric-cli` globally installed instead.